### PR TITLE
Fixed issue #17560: Layout issue when setting user rights.

### PR DESCRIPTION
**Pull request by gabrieljenik**
Scroll added to table using Bootstrap 3 classes.

### CODE REVIEW

File: application/views/userManagement/partial/editpermissions.php [hunk 1]
```diff
@@ -6,10 +6,10 @@
 ?>
 
 <?=TbHtml::formTb(null, App()->createUrl('userManagement/saveUserPermissions'), 'post', ["id"=>"UserManagement--modalform"])?>
-<div class="modal-body selector--edit-permissions-container">
+<div class="modal-body selector--edit-permissions-container table-responsive">
     <div class="container-center">
             <input type='hidden' name='userid' value='<?php echo (isset($oUser) ? $oUser->uid : '');?>' />
-            <table id='UserManagement--userpermissions-table' class='activecell table table-striped'>
+            <table id='UserManagement--userpermissions-table table' class='activecell table table-striped'>
```
Comment (Shnoulle) [inline application/views/userManagement/partial/editpermissions.php:12]: it's id : remove ` table`

it's already in class

File: application/views/userManagement/partial/editpermissions.php [hunk 2]
```diff
@@ -6,10 +6,10 @@
 ?>
 
 <?=TbHtml::formTb(null, App()->createUrl('userManagement/saveUserPermissions'), 'post', ["id"=>"UserManagement--modalform"])?>
-<div class="modal-body selector--edit-permissions-container">
+<div class="modal-body selector--edit-permissions-container table-responsive">
```
Comment (Shnoulle) [inline application/views/userManagement/partial/editpermissions.php:9]: table-responsice are awfull for long size tabme, make table unusuable. I send a video on small screen

**Review by Shnoulle — CHANGES_REQUESTED** [review]
1. Error on i
2. Solution is worst the bu in UX

**Comment by gabrieljenik** [comment]
Added a full screen modal class

**Comment by Shnoulle** [comment]
> Added a full screen modal class

on a binary file :sob:

**Review by Shnoulle — APPROVED** [review]

**Comment by Shnoulle** [comment]
1. Why not in adminbasic with a specific class name dialog-fullwidth for example
2. `width: auto; margin-left: 1em;margin-right: 1em;` seems great too
3. Seems important not needed (and it's better to don't have it) : adminbasic came after bootsrap.

Capture d’écran du 2023-05-03 09-48-27

**Review by Shnoulle — APPROVED** [review]
Thanks ! 😍

**Pull request merged**